README.md

```diff
@@ -30,6 +30,41 @@ This is particularly important in an environment where static IP allocations are
 
 Not anymore with Cleaner's self-healing magic! If you rely on DANM's powerful IPAM to manage the IP addresses in your cluster, and especially if you also use static allocations we strongly suggest installing Cleaner in your long-running, production environment.
 
+### Static CNIs 
+#### Calico
+In some cases calico IPAM cannot release registered IPs, like when a whole cluster goes down. On the long run this could cause calico to deplete its configured IP allocation pools, thus preventing new pods from aquiring IPs.
+
+Cleaner supports cleaning up occupied calico IPs while cleaning up dangling DanmEps.
+##### Install Steps
+1. Clone danm-utils project 
+2. Gather used calico version on your cluster
+3. Set gathered version as `CALICOCTL_VERSION` docker build argument in `scm/build/Dockerfile` to have cleaner use same calicoctl version as calico in your cluster
+4. Build cleaner image with `build_cleaner.sh`
+5. Create configmaps needed for calicoctl
+   - When connection to an etcd datastore
+     - calicoctl config
+       ```shell script
+       kubectl create configmap calico-config --from-file=/etc/calico/calicoctl.cfg
+       ```
+       [Example config](https://docs.projectcalico.org/getting-started/clis/calicoctl/configure/etcd#example-configuration-file)
+     - etcd certificates
+       ```shell script
+       kubectl create configmap etcd-certs-configmap --from-file=/etc/etcd/ssl
```

Comment:
secret
and only from the client cert we need to connect to the datastore

Reply:
Updated to use secret and selective client certificates